apps/cowswap-frontend/src/common/constants/common.ts

```diff
@@ -7,3 +8,6 @@ export const HIGH_FEE_WARNING_PERCENTAGE = new Percent(1, 10)
 export const SAFE_COW_APP_LINK = 'https://app.safe.global/share/safe-app?appUrl=https%3A%2F%2Fswap.cow.fi&chain=eth'
 
 export const MAX_ORDER_DEADLINE = ms`182d` + ms`12h` // 6 months, matching backend's https://github.com/cowprotocol/infrastructure/blob/901ed8e2fe3ea57956585f107bdd7539c2e7d3d1/services/Pulumi.yaml#L15
+
+// Use a 150K gas as a fallback if there's issue calculating the gas estimation (fixes some issues with some nodes failing to calculate gas costs for SC wallets)
```

Comment:
great, and not from this PR, but should we emit also a Sentry error?
I feel we just catch and silently handle the error

Reply:
I think it's not an exception but expected behaviour.
Some tokens don't allow approvals to amount higher than account balance.
Because of it we have this fallback

Reply:
<img width="760" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/4940aa98-53d1-49a5-bbcb-6bc5c4dd3540">

Reply:
Maybe we can change a bit the comment to reflect that. 


What about:

Fallback: Attempt to set an approval for the maximum wallet balance (instead of the MaxUint256). Some tokens revert if you try to use more than what you have.

Reply:
https://github.com/cowprotocol/cowswap/pull/3756/commits/a40768da4b3be0d8b028936c97d4b09340203126

Changed the comment, replaced the const duplications and deleted one legacy hook